tests/xtimer_msg/main.c

```diff
@@ -38,13 +38,14 @@ struct timer_msg {
     msg_t msg;
 };
 
-struct timer_msg msg_a = { .interval = 2*(1000000), .text = "Hello World", };
-struct timer_msg msg_b = { .interval = 5*(1000000), .text = "This is a Test" };
+struct timer_msg msg_a = {  .interval = (2 * US_PER_SEC),
```

Comment:
2 spaces after `{`, I think 1 is enough ;)

Reply:
yeah, I aligned `.interval` and `.text` below on tabs with size 4, but typically we only have 1 space here, will adapt.